content_scripts/vimium_frontend.coffee

```diff
@@ -563,7 +563,8 @@ findAndFollowLink = (linkStrings) ->
 
     linkMatches = false
     for linkString in linkStrings
-      if (link.innerText.toLowerCase().indexOf(linkString) != -1)
+      if (link.innerText.toLowerCase().indexOf(linkString) != -1 || link.value &&
+          link.value.indexOf(linkString) != -1)
         linkMatches = true
```

Comment:
Thanks, @sco-tt.

Could we get rid of the unnecessary parentheses here?  (I know they were there before, but we can tidy the code up as we touch it.)

Also, here (and below) the dependency on the precedence of `||` and `&&` should be made explicit.  Here, it can just be `link.value?.indexOf...`, below it will need parentheses.